packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -0,0 +1,301 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'dart:ui' as ui;
+
+import 'package:flutter/painting.dart';
+
+import 'object.dart';
+import 'stack.dart';
+
+// Data used by the DebugOverflowIndicator to manage the regions and labels for
+// the indicators.
+class _OverflowRegionData {
+  const _OverflowRegionData({
+    this.rect,
+    this.label = '',
```

Comment:
This may be something we should add to the style guide one way or the other (and maybe to our wishlist for dartfmt while we're at it, since this is one of those things it won't touch today)... right now, in the Flutter codebase we use `:` for default values of named arguments. We could move everything over to `=` (which would have the advantage of matching how default values are specified for positional optional arguments), but until we do we should probably keep the consistency going...